app/components/primitive-select/template.hbs

```diff
@@ -1,8 +1,8 @@
 <select class="form-control" {{action 'update' on="change"}} name={{name}} data-stripe={{data-stripe}}>
     {{#if prompt}}
-      <option disabled>{{prompt}}</option>
+      <option disabled selected>{{prompt}}</option>
```

Comment:
This is good for our current use cases (it will always select the prompt), but doesn't allow us to have a primitive-select that has an initially selected value.

To do that, the template would look like:

```handlebars
    <select class="form-control" {{action 'update' on="change"}} name={{name}} data-stripe={{data-stripe}}>      
      {{#if prompt}}
        <option disabled selected={{if (not selected) true null}}>{{prompt}}</option>
      {{/if}}
      {{#each items as |item|}}
        <option selected={{if (eq selected item) true null}}>{{item}}</option>
      {{/each}}
    </select>
```

JSBin Demo: http://emberjs.jsbin.com/nayaho/edit?html,js